utils.js

```diff
@@ -65,7 +65,7 @@ async function fetchNotarizationFromEndPoint (endPoint, txHex) {
     }
     return null
   } catch (e) {
-    return e
+    return e.response.data
```

Comment:
is this consistent with all browsers and exception types?

Reply:
Notary is returning a status code 500 on rejected txs (which shouldn't be the case imho). Axios handles 500s the same way so i think it's safe to go with this. We can also throw the error object alone but you would have to do the same in the upper level when handling the error.

Reply:
i mean response.data is semantic and valid? we don't want to see a null ptr error on different browsers / OSes

Reply:
It should be as long as the request returns a body which i think is the case for notary. I was not involved at that level in the project so i can't tell for sure. We can always throw `e` alone if we want to play it safe.

Reply:
is it related to response body? I was talking about the exception is it standard RFC spec that error.response.data always exists?